typescript-template-literals/main.ts

```diff
@@ -0,0 +1,22 @@
+const number = 9200;
```

Comment:
Starting here update this ts ( typescript ) file from js to modern ts. Then resubmitt.
**Remember** if it isn't specified still do so as it is a requirement for a correct submission.

Reply:
![Screen Shot 2024-02-05 at 16 54 51 PM](https://github.com/AdamZdravkovic/m1-0124-code-solutions/assets/87776374/9b68bd40-1fd0-43cd-b08c-1377b4a2c90a)